Pinta.Core/ImageFormats/OraFormat.cs

```diff
@@ -291,6 +287,18 @@ private BlendMode StandardToBlendMode (string mode)
 					return BlendMode.Screen;
 				case "svg:xor":
 					return BlendMode.Xor;
+                case "svg:hard-light":
+                    return BlendMode.HardLight;
+                case "svg:soft-light":
+                    return BlendMode.SoftLight;
+                case "svg:color":
+                    return BlendMode.Color;
+                case "svg:luminosity":
+                    return BlendMode.Luminosity;
+                case "svg:hue":
+                    return BlendMode.Hue;
+                case "svg:saturation":
+                    return BlendMode.Saturation;
 				default:
 					Console.WriteLine ("Unrecognized composite-op: {0}, using Normal.", mode);
```

Comment:
Since there are now cases where users could try to open a .ora file with an unrecognized blend mode, maybe it would be good to show a dialog here instead of printing to the console?